source/API_Reference/Event_Webhook/event.md

```diff
@@ -0,0 +1,1043 @@
+    {
+      "to": [
+        {
+          "email": "example@example.com"
+        }
+      ],
+      "subject": "Hello, World!",
+      "custom_args": {
+        "userid": "1123"
+      }
+    }
+  ],
+  "from": {
+    "email": "from_address@example.com"
+  },
+  "content": [
+    {
+      "type": "text/plain",
+      "value": "Hello, World!"
+    }
+  ]
+}
+{% endcodeblock %}
+
+The Event Webhook response:
+
+{% codeblock lang:json %}
+[
+  {
+    "userid": "1123"
+  }
+]
+{% endcodeblock %}
+
+- <a name="marketingcampaignid"></a>`marketing_campaign_id`
+- <a name="marketingcampaignname"></a>`marketing_campaign_name`
+
+For emails sent through our Marketing Campaigns feature, we will add Marketing Campaigns specific params to the Event Webhook. Both `marketing_campaign_name` and `marketing_campaign_id` are displayed as unique arguments in the event data.
+
+{% anchor h3 %}
+Example event from a standard (non-A/B test) campaign send:
+  {% endanchor %}
+
+{% codeblock lang:json %}
+  {
+    "category": [],
+    "email": "email@example.com",
+    "event": "processed",
+    "marketing_campaign_id": 12345,
+    "marketing_campaign_name": "campaign name",
+    "post_type": "event",
+    "sg_event_id": "sendgrid_internal_event_id",
+    "sg_message_id": "sendgrid_internal_message_id",
+    "sg_user_id": 12345,
+    "smtp-id": "",
+    "timestamp": 1442349428
+  }
+{% endcodeblock %}
+
+{% anchor h3 %}
+Example event from an A/B Test:
+{% endanchor %}
+
+ `marketing_campaign_version` will be displayed in the event data for emails sent as part of an A/B Test. The value for `marketing_campaign_version` are returned as `A`, `B`, `C`, etc.
+
+{% codeblock lang:json %}
+{
+  "category": [],
+  "email": "tadpole_0010@stbase-018.sjc1.sendgrid.net",
+  "event": "processed",
+  "marketing_campaign_id": 23314,
+  "marketing_campaign_name": "unique args ab",
+  "marketing_campaign_version": "B",
+  "marketing_campaign_split_id": 13471,
+  "post_type": "event",
+  "sg_event_id": "qNOzbkTuTNCdxa1eXEpnXg",
+  "sg_message_id": "5lFl7Fr1Rjme_EyzNNB_5A.stfilter-015.5185.55F883172.0",
+  "sg_user_id": 939115,
+  "smtp-id": "<5lFl7Fr1Rjme_EyzNNB_5A@stismtpd-006.sjc1.sendgrid.net>",
+  "timestamp": 1442349848
+}
+{% endcodeblock %}
+
+{% anchor h3 %}
+Example event from the winning phase of an A/B Test:
+{% endanchor %}
+
+{% codeblock lang:json %}
+{
+  "category": [],
+  "email": "tadpole_0001@stbase-018.sjc1.sendgrid.net",
+  "event": "delivered",
+  "marketing_campaign_id": 23314,
+  "marketing_campaign_name": "unique args ab",
+  "post_type": "event",
+  "response": "250 Ok ",
+  "sg_event_id": "X2M1IUfMRhuAhWM0CbmFqQ",
+  "sg_message_id": "fPJrJPIRTxC_obpgfTy74w.stfilter-015.5185.55F883564.0",
+  "sg_user_id": 12345,
+  "smtp-id": "",
+  "timestamp": 1442349911
+}
+{% endcodeblock %}
+
+{% anchor h3 %}
+Legacy Marketing Email Unsubscribes
+{% endanchor %}
+
+For emails sent through our Legacy Marketing Email tool, unsubscribes will look like the following example:
```

Comment:
For emails sent through our Legacy Marketing Email tool, unsubscribes ~~will~~ look like the following example: